met-api/src/met_api/models/language.py

```diff
@@ -0,0 +1,18 @@
+"""Language model class.
+
+Manages the Language
+"""
+from __future__ import annotations
+from .base_model import BaseModel
+from .db import db
+
+
+class Language(BaseModel):
+    """Definition of the Language entity."""
+
+    __tablename__ = 'language'
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String(50), nullable=False)  # eg. English, French etc
+    code = db.Column(db.String(2), nullable=False)  # eg. en, fr etc
```

Comment:
```suggestion
    code = db.Column(db.String(2), nullable=False, unique=True)  # eg. en, fr etc
```

We want to make sure that the language `code` is unique, as per the ticket. For example, we don't want users to be able to enter "fr" twice. Is there a better way to do this other than specifying it in the DB model?